src/index.js

```diff
@@ -1,6 +1,19 @@
 import React from "react";
```

Comment:
please separate redux store with index.js file and also try to use `thunk` and `applymiddleware`